### ISSUE TITLE
boot2docker-vm and virtualbox 5.0.2 do not seem to work together 

### ISSUE THREAD

**Issue by paetling** [gamechanger/dusty#533]
Get this error from dusty: 

`ERROR: Command '['boot2docker', 'start']' returned non-zero exit status 1`

Get this error from boot2docker:
```
(dusty_venv)Alexs-MacBook-Pro:~ dayman$ boot2docker start -vvv
Boot2Docker-cli version: v1.8.0
Git commit: 9a26066

  WARNING: The 'boot2docker' command line interface is officially deprecated.

  Please switch to Docker Machine (https://docs.docker.com/machine/) ASAP.

  Docker Toolbox (https://docker.com/toolbox) is the recommended install method.

2015/08/20 10:44:21 executing: VBoxManage showvminfo boot2docker-vm --machinereadable
2015/08/20 10:44:21 executing: VBoxManage guestproperty set boot2docker-vm /VirtualBox/GuestAdd/SharedFolders/MountPrefix /
2015/08/20 10:44:21 executing: VBoxManage guestproperty set boot2docker-vm /VirtualBox/GuestAdd/SharedFolders/MountDir /
2015/08/20 10:44:21 executing: VBoxManage sharedfolder add boot2docker-vm --name Users --hostpath /Users --automount
VBoxManage: error: Shared folder named 'Users' already exists
VBoxManage: error: Details: code VBOX_E_OBJECT_IN_USE (0x80bb000c), component SessionMachine, interface IMachine, callee nsISupports
VBoxManage: error: Context: "CreateSharedFolder(Bstr(name).raw(), Bstr(hostpath).raw(), fWritable, fAutoMount)" at line 1020 of file VBoxManageMisc.cpp
2015/08/20 10:44:21 executing: VBoxManage setextradata boot2docker-vm VBoxInternal2/SharedFoldersEnableSymlinksCreate/Users 1
2015/08/20 10:44:21 executing: VBoxManage startvm boot2docker-vm --type headless
Waiting for VM "boot2docker-vm" to power on...
VM "boot2docker-vm" has been successfully started.
error in run: Failed to start machine "boot2docker-vm": exit status 1
```

VirtualMachine 4.3 works fine. I am going to downgrade for now.

**Comment by paetling** [comment]
More info:
```
Alexs-MacBook-Pro:~ dayman$ boot2docker version
Boot2Docker-cli version: v1.8.0
Git commit: 9a26066
Alexs-MacBook-Pro:~ dayman$ boot2docker ssh
                        ##         .
                  ## ## ##        ==
               ## ## ## ## ##    ===
           /"""""""""""""""""\___/ ===
      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~~ ~ /  ===- ~~~
           \______ o           __/
             \    \         __/
              \____\_______/
 _                 _   ____     _            _
| |__   ___   ___ | |_|___ \ __| | ___   ___| | _____ _ __
| '_ \ / _ \ / _ \| __| __) / _` |/ _ \ / __| |/ / _ \ '__|
| |_) | (_) | (_) | |_ / __/ (_| | (_) | (__|   <  __/ |
|_.__/ \___/ \___/ \__|_____\__,_|\___/ \___|_|\_\___|_|
Boot2Docker version 1.8.1, build master : 7f12e95 - Thu Aug 13 03:24:56 UTC 2015
Docker version 1.8.1, build d12ea79
```

**Comment by paetling** [comment]
Error on boot2docker startup through the VirtualBox gui:
![image](https://cloud.githubusercontent.com/assets/3281726/9387449/8fbcb650-472e-11e5-9c22-bdf7034b89aa.png)

**Comment by thieman** [comment]
This is a known issue with VirtualBox, and looks to be limited to the `5.0.1` and `5.0.2` releases. https://www.virtualbox.org/ticket/14412

They've fixed it in master, so this should be resolved as of `5.0.3`. In the meantime, you can work around this by either downgrading to `5.0.0` or `4.3.30`.

**Comment by thieman** [comment]
VirtualBox 5.0.4 is out, can anyone try reproducing with that version and see if it's fixed?

**Comment by thieman** [comment]
Closing, fix is to use VirtualBox 5.0.4+
